dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/config/validation/AbstractConfigurationValidator.java

```diff
@@ -1004,8 +1017,15 @@ private boolean hasFeature(ConfigurationFile file, String featureName) {
      */
     private ValidationContext getCommonParent(ValidationContext context, String featureName) {
         ValidationContext toplevl = context;
-        while (!hasFeature(toplevl.getConfigFile(), featureName)) {
-            toplevl = toplevl.getParent();
+        ConfigurationFile configFile = toplevl.getConfigFile();
+        if (configFile != null) { // Protect against null.
+            while (!hasFeature(configFile, featureName)) {
```

Comment:
Can it just be: while (configFile != null && !hasFeature(configFile, featureName))?